apps/store/extensions/app/social-reviews/themes/store/partials/stream.hbs

```diff
@@ -35,9 +35,16 @@
 				{{> activity}}
 			{{/each}}
 		{{else}}
-			<p class="com-first-review">
-				Be the first one to review!
-			</p>
+            {{#if myReview}}
+                    <div class="com-first-review message message-success" style="display: block;">
+                        <h4><i class="fw fw-success"></i>You are the first to review this {{type}}</h4>
+                    </div>
+            {{else}}
+            <div class="com-first-review message message-info">
+                <h4><i class="fw fw-info"></i>Be the first one to review!</h4>
+                <p>Tell others what you think about this {{type}}</p>
```

Comment:
Thanks for the review, I have changed the word embedding to use `t` helper with the following commit 
[674d41293ed0dbdfaf43ac6bba5f8b042969f88d](https://github.com/wso2/carbon-store/pull/643/commits/674d41293ed0dbdfaf43ac6bba5f8b042969f88d)